sheets/coupon_assign_api.py

```diff
@@ -110,10 +105,12 @@ def parse_raw_data(cls, row_index, raw_row_data):
             SheetRowParsingException: Raised if the row could not be parsed
         """
         try:
+            raw_email = item_at_index_or_none(raw_row_data, 1)
+            email = None if raw_email is None else raw_email.lower()
```

Comment:
What about the empty string?
Should it not be `email = None if not raw_email else raw_email.lower()`